addons/context_actions/functions/fnc_canUnloadViV.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Checks if the given objects list contains ViV cargo or carriers that can be unloaded.
+ *
+ * Arguments:
+ * N: Objects <OBJECT>
+ *
+ * Return Value:
+ * Can Unload Vehicles <BOOL>
+ *
+ * Example:
+ * [_object] call zen_context_actions_fnc_canUnloadViV
+ *
+ * Public: No
+ */
+
+_this findIf {
+    !(getVehicleCargo _x isEqualTo []) || {!isNull isVehicleCargo _x}
+} != -1
```

Comment:
I think this condition is short enough to be a single line.

Reply:
a3963ff